frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -356,15 +356,56 @@ machinery-Machinery--search-placeholder =
 
 
 ## Machinery Translation
-## Shows a specific translation from machinery
+## Shows a specific translation from machinery.
 
 machinery-Translation--copy =
     .title = Copy Into Translation
 
-machinery-Translation--number-occurrences =
+
+## Machinery Translation Memory
+## Shows the translation source from Pontoon's memory.
+
+machinery-TranslationMemory--pontoon-homepage =
+    .title = Pontoon Homepage
+machinery-TranslationMemory--translation-memory = Translation memory
+machinery-TranslationMemory--number-occurrences =
     .title = Number of translation occurrences
 
 
+## Machinery Google Translation
+## Shows the translation source from Google Translate.
+
+machinery-GoogleTranslation--visit-google =
+    .title = Visit Google Translate
+
+
+## Machinery Microsoft Translation
+## Shows the translation source from Microsoft Translation.
+machinery-MicrosoftTranslation--visit-bing =
+    .title = Visit Bing Translate
+
+
+## Machinery Microsoft Terminology
+## Shows the translation source from Microsoft Terminology.
+machinery-MicrosoftTerminology--visit-microsoft =
+    .title = Visit Microsoft Terminology Service API.
+    © 2018 Microsoft Corporation. All rights reserved.
```

Comment:
I think this needs to be indented to align with the content on the previous line. Is that right @Pike?

Reply:
It just needs to be indented. I'd recommend making it a proper multi-line string, though:

```
## Shows the translation source from Microsoft Terminology.
machinery-MicrosoftTerminology--visit-microsoft =
    .title =
        Visit Microsoft Terminology Service API.
        © 2018 Microsoft Corporation. All rights reserved.
```